### DISCUSSION TITLE
password 'ros' not working

### DISCUSSION THREAD

**Discussion by srt9969** [rern/rAudio#741] [General]
Hi, I'm new to rAudio, and not particularly techy either - although I can follow instructions.
So, I understood from the readme that if i ssh pi@[myPi-IP], the password is 'ros'. Well, apparently not on my pi it isn't. 
Can anyone cast any light for me please?
RPi 3B+ , latest rAudio version, HiFiBerry digi+

Thank you...

**Comment by R101G** [comment]
@srt9969 
User is "root" not "pi"

**Comment by srt9969** [comment]
Ah, that does it! Told you I was a noob...
Thank you!
